src/dmd/e2ir.d

```diff
@@ -6362,6 +6454,65 @@ elem* elAssign(elem* e1, elem* e2, Type t, type* tx)
     return e;
 }
 
+
+/**********************************
+ * Assign OPstrctor in `e2` it to `e1`.
```

Comment:
This sentence makes no sense.

Reply:
I have no idea what this function does.

Reply:
I moved it to `cgelem.d` where it makes more sense. Check it now, it's a tree optimization.